picr22-enne-1-voistlust/segment_module/Segment.egg-info/PKG-INFO

```diff
@@ -0,0 +1,3 @@
+Metadata-Version: 2.1
```

Comment:
Unneeded file. add to git ignore and remove.

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it